integration/code/query_postgres_runtime_data_connector.py

```diff
@@ -5,7 +5,9 @@
 
 CONNECTION_STRING = """postgresql+psycopg2://postgres:@localhost/test_ci"""
 load_data_into_database(
-    "taxi_data", "../fixtures/test_data/reports/yellow_tripdata_sample_2019-01.csv"
+    "taxi_data",
+    "../fixtures/test_data/reports/yellow_tripdata_sample_2019-01.csv",
+    CONNECTION_STRING,
 )
```

Comment:
I ended up doing the same thing :) This fixed the circular imports error I was getting